packages/sdks/mitosis.config.cjs

```diff
@@ -786,6 +786,19 @@ module.exports = {
             },
           },
         }),
+        () => ({
+          code: {
+            post: (code) => {
+              if (
+                code.includes('BlocksWrapper') ||
+                code.includes('EnableEditor')
+              ) {
+                code = code.replace('onPress', 'onClick');
+              }
+              return code;
```

Comment:
a comment on why we need to do this would be good

Reply:
yup! added